shell/platform/embedder/embedder.cc

```diff
@@ -93,8 +93,19 @@ static bool IsOpenGLRendererConfigValid(const FlutterRendererConfig* config) {
 
   if (SAFE_ACCESS(open_gl_config, make_current, nullptr) == nullptr ||
       SAFE_ACCESS(open_gl_config, clear_current, nullptr) == nullptr ||
-      SAFE_ACCESS(open_gl_config, present, nullptr) == nullptr ||
-      SAFE_ACCESS(open_gl_config, fbo_callback, nullptr) == nullptr) {
+      SAFE_ACCESS(open_gl_config, present, nullptr) == nullptr) {
+    return false;
+  }
+
+  bool fbo_callback_exists =
+      SAFE_ACCESS(open_gl_config, fbo_callback, nullptr) != nullptr;
+  bool fbo_with_frame_info_callback_exists =
+      SAFE_ACCESS(open_gl_config, fbo_with_frame_info_callback, nullptr) !=
+      nullptr;
+  // only one of these callbacks must exist.
+  if (fbo_callback_exists == fbo_with_frame_info_callback_exists) {
+    LOG_EMBEDDER_ERROR(kInternalInconsistency,
```

Comment:
`LOG_EMBEDDER_ERROR` is only used right before returning control to the caller. `kInternalInconsistency` isn't actually going to be returned to the caller here. Just get rid of this line as the result of `IsRendererValid` is used to generate the error later.

Reply:
Done